src/collections/members/yash-vardhan-grumpyyash/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+name: Yash Vardhan Grumpyyash
```

Comment:
name: Yash Vardhan

Reply:
ok will apply changes

Reply:
Perhaps, Yash (Grumpy) Vardhan is more appropriate....     😉